CMakeLists.txt

```diff
@@ -194,7 +194,9 @@ set(CMAKE_C_FLAGS_DEBUGOPT "")
 
 target_compile_options(${PROJECT_NAME} PRIVATE -pedantic -std=gnu99 -Wall -Werror -Wimplicit -Wunused -Wcomment -Wchar-subscripts
         -Wuninitialized -Wshadow -Wcast-qual -Wcast-align -Wwrite-strings -Wno-deprecated-declarations -Wno-unknown-pragmas -Wformat-security
-        -Wno-missing-braces -fvisibility=hidden -DS2N_EXPORTS)
+        -Wno-missing-braces -fvisibility=hidden -DS2N_EXPORTS
+        -Wa,--noexecstack # make assembler output compatible with SELinux
```

Comment:
Should these flags be added to the Make build as well?

https://github.com/aws/s2n-tls/blob/b63d60b72396113014720ccda19eaffa559f6a52/s2n.mk#L45-L49

Reply:
Probably. I have only used this module as part of aws-crt-cpp which uses cmake. I can blindly add the setting into s2n.mk but don't have an easy way to test the change.

Reply:
I tested the compile flag in `s2n.mk` and verified execstack returns a non-exec bit.

```
──> execstack -q bin/s2nc                                                                                                                                                                                        130 ↵ ──(Thu,Apr01)─┘
X bin/s2nc
```

If you want to push the change I can merge.

Reply:
lol, sorry, that output was the initial "control" build. Here is the good one:

```
──> execstack -q bin/s2nc                                                                                                                                                                                      130 ↵ ──(Thu,Apr01)─┘
- bin/s2nc
```